src/rgw/rgw_acl_types.h

```diff
@@ -181,12 +181,15 @@ class ACLGranteeType
 protected:
   __u32 type;
 public:
-  ACLGranteeType() : type(ACL_TYPE_UNKNOWN) {}
+  ACLGranteeType(ACLGranteeTypeEnum t = ACL_TYPE_UNKNOWN) : type(t) {}
   virtual ~ACLGranteeType() {}
```

Comment:
there are no virtual functions. why do we need a virtual dtor?

Reply:
yeah, i noticed that yesterday and added a commit to remove it. i had cleaned up all of the virtuals in rgw_acl.h, but missed that one in rgw_acl_types.h